openid-4-verifiable-presentations-1_0.md

```diff
@@ -544,7 +544,7 @@ If the Verifier responds with any HTTP error response, the Wallet MUST terminate
 
 # Response {#response}
```

Comment:
It feels like this whole section is not consistent (i.e., would need more changes) if we want to make VP Token optional.

Reply:
where can i find how the proposal on how the response with a new query language will look like? without that it's very hard for me to make sense of the changes proposed in the vp token section.

Reply:
See https://github.com/openid/OpenID4VP/issues/178 and the vp token examples, e.g.:

<img width="900" alt="Screenshot 2024-09-25 at 09 22 34" src="https://github.com/user-attachments/assets/2bf87d8f-328f-48a5-aef9-097bba398913">